### PR TITLE
doc(release): Release notes for collect 23.04

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [x] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 1]
```diff
@@ -455,6 +455,18 @@ Release date: `April 26, 2023`
 
 ## Centreon Collect
 
+### 23.04.14
+
+Release date: `July 7, 2024`
```
Comment (sfarouq-ext) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:460]: Hello is it July 17 ?

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 2]
```diff
@@ -453,6 +453,18 @@ Release date: `April 26, 2023`
...
 ## Centreon Collect
 
+### 23.04.14
+
+Release date: `July 17, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Broker] If a metric can't be inserted in the database, the metric is dropped with an error log but Broker continues to work, and others metrics are still inserted.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:463]:
```suggestion
- [Broker] Fixed an issue that occurred when inserting too large metrics into the database, which resulted in an error log.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx [hunk 3]
```diff
@@ -453,6 +453,18 @@ Release date: `April 26, 2023`
...
+- [Broker] Fixed an issue while trying to Connect to MySQL 8.0.35.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx:464]:
```suggestion
- [Broker] Added support for the default connection method to MySQL 8.0.35.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: versioned_docs/version-23.04/releases/centreon-os.mdx (hunk 1 above)
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:460]:
```suggestion
Release date: `July 17, 2024`
```

File: versioned_docs/version-23.04/releases/centreon-os.mdx [hunk 4]
```diff
@@ -455,6 +455,18 @@ Release date: `April 26, 2023`
...
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Broker] If a metric can't be inserted in the database, the metric is dropped with an error log but Broker continues to work, and others metrics are still inserted.
+- [Broker] Fixed an issue while trying to Connect to MySQL 8.0.35.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-os.mdx:466]:
```suggestion
- [Broker] Fixed an issue that occurred when inserting too large metrics into the database, which resulted in an error log.
- [Broker] Added support for the default connection method to MySQL 8.0.35.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by sfarouq-ext — APPROVED** [review]

**Pull request merged**